tests/features/api/reverse.feature

```diff
@@ -11,3 +11,23 @@ Feature: Reverse geocoding
          | ID | country
          | 0  | Deutschland
 
+    @Tiger
+    Scenario: TIGER house number
+        Given the request parameters
+          | addressdetails
+          | 1
+        When looking up jsonv2 coordinates 40.6863624710666,-112.060005720023
+        # Then exactly 1 result is returned
+        # Then result addresses contain
+        # | ID | house_number | road               | postcode | country_code
+        # | 0  | 7094         | Kings Estate Drive | 84128    | us
```

Comment:
This one only works against xml format, so use `When looking up coordinates` above. Also you should explicitly test for Tiger results with `And result 0 has not attributes osm_id,osm_type`.

The tests below don't do anything.